src/frontend/web_application/.eslintrc

```diff
@@ -1,19 +1,10 @@
 {
   "extends": "airbnb",
   "parser": "babel-eslint",
-  "ecmaFeatures": {
-    "modules": true
-  },
-  "parserOptions": {
-    "ecmaVersion": 6,
-    "ecmaFeatures": {
-      "experimentalObjectRestSpread": true
-    }
-  },
+  "root": true,
   "env": {
     "browser": true,
-    "es6": true,
-    "jasmine": true,
```

Comment:
> it fails on jasmine dependency

What do you mean? I haven't noticed any error related to this. 
Also Jest is moving away from Jasmine, they have their own assertion methods now.